src/main/scala/com/databricks/spark/sql/perf/Benchmarkable.scala

```diff
@@ -35,10 +39,16 @@ trait Benchmarkable {
       includeBreakdown: Boolean,
       description: String = "",
       messages: ArrayBuffer[String],
-      timeout: Long): BenchmarkResult = {
+      timeout: Long,
+      forkThread: Boolean = true): BenchmarkResult = {
+    logger.info(s"$this: benchmark")
     sparkContext.setJobDescription(s"Execution: $name, $description")
     beforeBenchmark()
-    val result = runBenchmark(includeBreakdown, description, messages, timeout)
+    val result = if (forkThread) {
+      doBenchmark(includeBreakdown, description, messages)
+    } else {
+      runBenchmarkForked(includeBreakdown, description, messages, timeout)
```

Comment:
This is mostly a cosmetic issue and I will be happy to revert it. When we fork. we break the progress reporting in databricks, and we do not have progress bars and other information. Since it is also accessible through the UI, I am tempted to revert this change.